pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/CreateClinicalDummyKitRoute.java

```diff
@@ -52,7 +53,8 @@ public static void addCollaboratorSampleId(String tissueId, DDPInstance ddpInsta
         String collaboratorParticipantId = KitRequestShipping
                 .getCollaboratorParticipantId(ddpInstance.getBaseUrl(), ddpInstance.getDdpInstanceId(), ddpInstance.isMigratedDDP(),
                         ddpInstance.getCollaboratorIdPrefix(), ddpParticipantId, shortId, "4");
-        updateTissue(tissueId, collaboratorParticipantId + "_T1");
+        int randomINT = new Random().nextInt(100000);
+        updateTissue(tissueId, collaboratorParticipantId + "_T" + randomINT);
```

Comment:
What are the consequences of a collision here? There's no guarantee that two calls to `nextInt()` will not return the same value.

Reply:
So the collaborator sample id should be unique, but since we are using it with `collaboratorParticipantId` , it is already different between tissues for 2 different participants, so the only case of that is when 2 dummy tissues being created for  the same participant get the same random int. We can add a check, but I think in that case they can just test again. I don't think this will happen